sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/DescribeTableSuite.scala

```diff
@@ -548,6 +547,7 @@ trait DescribeTableSuiteBase extends command.DescribeTableSuiteBase
                |""".stripMargin
           spark.sql(tableCreationStr)
           val viewType = if (isTemp) "TEMP VIEW" else "VIEW"
+          spark.sql(s"SET spark.sql.ansi.enabled = true")
```

Comment:
since ANSI is enabled by default, shall we test setting it as off?
also, let's test multiple sql conf changes instead of just one